docs/Developers/Building-Locally.md

```diff
@@ -1,18 +1,20 @@
 # Building Locally
 
-This is a guide to editing, building, running and deploying Unciv from code
-
-So first things first - the initial "No assumptions" setup to have Unciv run from-code on your computer!
+By the end of this guide, you will have Unciv running locally from code, so you can make changes and test them locally.
 
 ## With Android Studio
 
--   Install Android Studio - it's free and awesome! Be aware that it's a long download!
+-   Install [Android Studio](https://developer.android.com/studio) - it's free and awesome! Be aware that it's a long download!
 -   Install Git, it's the way for us to work together on this project. UI is optional, Android Studio has good Git tools built in :)
 -   Getting the code
-    -   Create a Github account, if you don't already have one
-    -   Fork the repo (click the "Fork" button on the top-right corner of https://github.com/yairm210/Unciv) - this will create a "copy" of the code on your account, at https://github.com/YourUsername/Unciv
-    -   Clone your fork with git - the location will be https://github.com/YourUsername/Unciv.git, visible from the green "Clone or download" button at https://github.com/YourUsername/Unciv
--   Load the project in Android Studio, Gradle will attempt the initial sync. If this is your first time with Android Studio, this may require you to accept the Android Build-tools licenses, which works differently on every device, so search for your OS-specific solution.
+    -   Create a [Github account](https://github.com/join), if you don't already have one
+    -   [Fork the repo](https://github.com/yairm210/Unciv/fork) - this will create a "copy" of the code on your account, at `https://github.com/<YourUsername>/Unciv`
+    -   Clone your fork with git - go to `https://github.com/<YourUsername>/Unciv` and click the green "Clone or download" button
+-   Load the project in Android Studio
+    - File -> New -> Project from Version Control -> GitHub
+    - Enter your GitHub username and password
+    - Select the repository and hit clone - The GitHub repo will be created as a new project in Android Studio.
```

Comment:
Wouldn't this clone the repo twice as you've already cloned it before in line 12?

Reply:
You're right, we end up not using the "clone with git" step above, good catch!